stake-pool/cli/src/main.rs

```diff
@@ -1158,14 +1159,24 @@ fn command_update(
 
     let validator_list = get_validator_list(&config.rpc_client, &stake_pool.validator_list)?;
 
-    let (mut update_list_instructions, final_instructions) =
+    let (mut update_list_instructions, final_instructions) = if fresh {
         spl_stake_pool::instruction::update_stake_pool(
             &spl_stake_pool::id(),
             &stake_pool,
             &validator_list,
             stake_pool_address,
             no_merge,
-        );
+        )
+    } else {
+        spl_stake_pool::instruction::update_stale_stake_pool(
+            &spl_stake_pool::id(),
+            &stake_pool,
+            &validator_list,
+            stake_pool_address,
+            no_merge,
+            epoch_info.epoch,
+        )
+    };
```

Comment:
I know we chatted about this new flag previously, but what do you think about making this flag something like `--stale-only` and flipping this conditional around?

I think that might be less confusing considering the existing command with just `--force` will still behave the same as before, while `--force --stale-only` would be a little more explicit way to say "do the update anyway but only update stale validators".

Reply:
Done. I didn't change the arg where `command_update()` is called before other commands tho so it defaults to `--stale-only false` for all the other commands